controller/backup_controller.go

```diff
@@ -393,6 +394,9 @@ func (bc *BackupController) reconcile(backupName string) (err error) {
 		return nil
 	}
 
+	// Remove the Backup Volume recurring jobs/groups information.
+	delete(backupInfo.Labels, types.VolumeRecurringJobInfoLabel)
```

Comment:
Recurring jobs/groups information is saving in the backup volume CR and backup volume configuration on remote backup target.
I think it should not be saved in every backup the recurring job or user created.

Reply:
It seems that `backupInfo` returned by the remote backup target contains both the backup volume info and the backup info. And you want to remove the redundancy, right?

Reply:
Yes

Reply:
I see. Actually, I am fine with storing the recurring job info in each Backup CRs since the recurring job config can be different between these backup CRs. If you just store the recurring job info in the BackupVolume CR only, it means you will blindly apply the latest recurring job config to any backups.

Reply:
Yes, after some discussions, now it blindly stores the latest recurring jobs of volume only.